liquidity_pool/src/liquidity.rs

```diff
@@ -345,23 +347,25 @@ pub trait LiquidityModule:
 
         self.borrow_position(borrow_position_nonce).clear();
 
-        let bp = self.get_base_precision();
         let lend_amount_to_return = (asset_amount * (&bp + &liquidation_bonus)) / bp;
+        let lend_amount_to_return_in_lend_token =
```

Comment:
Convert the amount to collateral tokens. Collateral tokens should be sent to the liquidator.